README.md

```diff
@@ -108,13 +108,13 @@ public interface BookRepository extends TarantoolRepository<Book, Long> {
 }
 ```
 
-The `@Tuple` annotation allows to specify the space name which schema will be used for forming the data tuples.
-You can add `@Tuple` on the `@Query` annotated method that specifies the space name for a specific stored function call.
-Also, you can add `@Tuple` annotation to the repository - it acts as a shortcut for specifying the tuple return format and space name for all custom methods
-in this repository. The `@Field` annotations provide custom names for the fields. It is necessary to have at least 
+The `@Tuple` annotation allows to specify the space name which schema will be used for forming the data tuples to sending
+and for mapping the incoming tuple into an object.
```

Comment:
Please proof-read the texts with Nick before merging.

Reply:
Yes, as soon as we finish the readme, I'll tag our new head of documentation, Nick is now on another team

Reply:
Added action point for readme review #85